### ISSUE TITLE
Lack of escaping in the track function

### ISSUE THREAD

**Issue by peterspicer-unfold** [woocommerce/pinterest-for-woocommerce#716]
### Describe the bug:
The code that sets up the pintrk() call in the JavaScript doesn't escape the product title, which means it will fail if the product title has an apostrophe in it.

### Steps to reproduce:
1. Create a product with an apostrophe in the title.
2. Visit that product page, observing the browser console.

### Expected behavior:
No console error and for the plugin to correctly track this with Pinterest.

### Actual behavior:
Uncaught SyntaxError: unexpected identifier 'Kursar'

This is related to the relevant snippet:
```
<script id='wc-add-to-cart-js-after'>
jQuery( function( $ ) {
	$( document.body ).on( 'added_to_cart', function( e, fragments, cart_hash, thisbutton ) {
		var quantity = document.querySelector( 'input.qty[name="quantity"]' ).value;
		pintrk( 'track', 'AddToCart', {
			'product_id': '23346',
			'product_name': 'Flowering cherry tree 'Kursar'',
			'value': 65 * quantity,
			'order_quantity': quantity,
			'currency': 'GBP'
		} );
	} );
} );
</script>
```

As you can see the product has Kursar in single quotes (which is the product name in the WooCommerce installation) but it is not escaped.

### Additional details:
Observed with 1.2.16.

Also appears to have been observed by another user: https://wordpress.org/support/topic/lack-of-escaping-in-javascript-causes-page-errors/

**Comment by message-dimke** [comment]
Hi, @peterspicer-unfold , here is the PR for it https://github.com/woocommerce/pinterest-for-woocommerce/pull/721

**Comment by peterspicer-unfold** [comment]
Thanks for the fix!

**Comment by ibndawood** [comment]
Hello @peterspicer-unfold. We've released `v1.2.19`, which will fix this issue. Please update the plugin and let us know if there are any issues.
